saist/util/rules.py

```diff
@@ -0,0 +1,39 @@
+import os
+import yaml
+
+class PromptRules:
+    RulesFile = "saist.rules"
+
+    @staticmethod
+    def apply_rules(prompt):
+        rules = PromptRules.load_rules()
+
+        override_prompt = rules.get("PROMPT_OVERRIDE")
+        if override_prompt:
+            #skip pre and post if override is present
```

Comment:
I don't think this should be the desired behaviour. If the user wants to declare a PRE and POST with their OVERRIDE, we should probably use it